bin/load-policy

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -eu
+
+demo_dir="$( cd "$(dirname "$0")/.." ; pwd -P )"
+
+# Initialize Conjur with the demo account
+cat << CONJURRC > "$demo_dir/cli/.conjurrc"
+appliance_url: http://conjur
+account: demo
+CONJURRC
+
+# Fetch admin API key
+api_key=$(docker-compose --project-directory "$demo_dir" \
```

Comment:
These `--project-directory $demo_dir` options are unnecessary, since the default project directory is the one where the `docker-compose.yml` file is found. So in effect you're setting it to the default :)

Reply:
👍